OpenQA/Isotovideo/Utils.pm

```diff
@@ -12,15 +12,18 @@ use bmwqemu;
 use autotest;
 use Try::Tiny;
 
-our @EXPORT_OK = qw(checkout_git_repo_and_branch checkout_git_refspec
-  handle_generated_assets load_test_schedule);
+our @EXPORT_OK = qw(git_rev_parse checkout_git_repo_and_branch
+  checkout_git_refspec handle_generated_assets load_test_schedule);
+
+sub git_rev_parse ($dirname) {
+    my $addsafe = q{HOME=$(mktemp -d os-autoinst-git.XXXXX) && git config --global --add safe.directory};
```

Comment:
Would this temp dir be cleaned up at some point? Especially since we seem to create always a new temp dir it would make sense. I suppose we could alternatively also just create a directory within the current working directory. That's where isotovideo already stores intermediate/temporary files and it would make sense to keep doing that. Since the worker normally cleans the "pool" directory you'd get the cleanup for free.

Reply:
Theoretically yes but os-autoinst does not know that openQA behaves like that … or at least shouldn't need to know ;) But please see https://github.com/os-autoinst/os-autoinst/pull/2104#discussion_r914934993 about the deletion. I will implement that proposal including the note from @perlpunk about the template plus the `--tempdir` option

Reply:
done